mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -269,13 +263,9 @@ function Data() {
     >
       <Tab key="data" label="Data">
         <Spacing mb={3} mt={3} />
-        <SimpleDataTable
-          columnFlexNumbers={ Array(columnHeaderSample.length).fill(1)}
-          columnHeaders={columnHeaderSample}
-          rowGroupData={rowGroupDataSample && [rowGroupDataSample]}
-        />
+        {dataEl}
       </Tab>
-      <Tab key="reports" label="Reports">
+      <Tab key="reports" label="Report">
```

Comment:
The designs have "Report" plural, which makes sense since there may be multiple "reports". Any reason you're undoing the change?

Reply:
Didn't catch it when copy/pasting. Will update